projects/js-packages/social-previews/src/mastodon-preview/post/actions/styles.scss

```
@@ -0,0 +1,26 @@
+@use "../../variables" as vars;
+
+.mastodon-preview__post-actions {
+	margin-top: 1rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	/* stylelint-disable-next-line scss/selector-no-redundant-nesting-selector */
+	& > div {
```

If above, then here too
```suggestion
	> div {
```